### PR TITLE
Update NewPackage() message for _jll

### PR DESCRIPTION
**Pull request by thofma**
Here my try at improving the message.

### CODE REVIEW

File: src/AutoMerge/new-package.jl [hunk 1]
```diff
@@ -245,7 +245,8 @@ function pull_request_build(api::GitHub.GitHubAPI,
                 this_pr_comment_fail = comment_text_fail(NewPackage(),
```
Comment (ericphanson) [inline src/AutoMerge/new-package.jl:245]: should this be `comment_text_pass`?

Comment (thofma) [reply src/AutoMerge/new-package.jl]: Oops. Fixed!

File: src/AutoMerge/util.jl [hunk 2]
```diff
@@ -59,19 +59,35 @@ end
...
+    if is_jll
+      result = string("Your `new _jll package` pull request met all of the ",
+                      "guidelines for auto-merging and is scheduled to ",
+                      "be merged in the next round."
+                      "\n\n",
+                      _comment_noblock(),
+                      _onepointzero_suggestion(suggest_onepointzero, version),
+                      "\n<!-- [noblock] -->")
+    else
+      result = string(is_jll ?
```
Comment (ericphanson) [inline src/AutoMerge/util.jl:73]: Hm, I think the `is_jll` logic is repeated, since it's both here and in the `if` statement above

**Review by ericphanson — APPROVED** [review]
Looks good to me! You'll need to bump the patch version in the Project.toml for the VersionVigilante check to pass. I'm not sure why the CI isn't running also.

**Comment by DilumAluthge** [comment]
CI is done via Bors.

bors try

**Comment by thofma** [comment]
Bumped the patch version.

**Comment by DilumAluthge** [comment]
bors try

If CI passes, I'll do a more thorough review.

**Comment by DilumAluthge** [comment]
bors try

**Comment by ericphanson** [comment]
Looks like a spurious failure due to network connectivity issues: https://travis-ci.com/github/JuliaRegistries/RegistryCI.jl/jobs/378129450#L184

**Comment by DilumAluthge** [comment]
bors try

**Comment by DilumAluthge** [comment]
bors merge

**Comment by ericphanson** [comment]
thanks @thofma! I'm sure you weren't the first confused by that message.

**Comment by thofma** [comment]
Always glad if I can turn my confusion into a PR.

**Pull request merged**